.github/workflows/auth-ci.yml

```
@@ -1,79 +1,87 @@
-          BUILDKIT_PROGRESS: plain
-        with:
-          context: .
-          file: ./Dockerfile
-          push: true
-          no-cache: true
-          provenance: false
-          tags: |
-            ${{ steps.vars.outputs.IMAGE }}:${{ steps.vars.outputs.TAG_VERSION }}
-            ${{ steps.vars.outputs.IMAGE }}:${{ steps.vars.outputs.TAG_LATEST }}
\ No newline at end of file
+name: CI (use Gradle version)
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)